tools/cli/adminElasticSearchCommands.go

```diff
@@ -214,6 +214,7 @@ func AdminDelete(c *cli.Context) {
 	rps := c.Int(FlagRPS)
 	ratelimiter := tokenbucket.New(rps, clock.NewRealTimeSource())
 
+	// #nosec
```

Comment:
@vancexu please review and provide comment.

Reply:
correct, no security concern here because file read as pure text.